ui-lib/src/main/java/co/electriccoin/zcash/ui/MainActivity.kt

```diff
@@ -139,6 +136,36 @@ class MainActivity : ComponentActivity() {
         }
     }
 
+    private suspend fun observeScreenSecurityFlag(screenSecurity: ScreenSecurity) {
```

Comment:
Nice refactoring of these into helper methods, as it makes the code much easier to follow